src/components/znkHeader/services/purchase.service.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * znkAnalyticsSrv
+ *
```

Comment:
what this service got to do with purchase.service ? also znkAnalyticsSrv should be znkHeaderSrv